bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/servicemonitor.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     matchNames:
       - {{ .Release.Namespace }}
   endpoints:
-    - port: http-metrics
+    - port: http
```

Comment:
You're completely right, see:
- https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/metrics-service.yaml#L38